lib/lookip.js

```diff
@@ -38,9 +46,14 @@ exports.mask = (cleanIpString) => {
   } else if (ip.isV6Format(cleanIpString)) {
     return ip.mask(cleanIpString, 'ffff:ffff:ffff:ffff:ffff:ffff:ffff:0');
   } else {
-    return null;
+    return cleanIpString;
```

Comment:
What case would this be where it is not a V4 or V6 IP and we still want to return it? Is this a just in case something weird comes through we "save" it?

Reply:
Yeah, my thought was if it's not a valid ip4/ip6 ... it's not identifiable info, so we can safely save it as-is.  (And use that to debug later on).  I could be convinced otherwise.